go/teams/chain.go

```diff
@@ -1970,6 +1983,20 @@ func (t *TeamSigChainPlayer) completeInvites(stateToUpdate *TeamSigChainState, c
 	}
 }
 
+func (t *TeamSigChainPlayer) obsoleteInvites(stateToUpdate *TeamSigChainState, roleUpdates chainRoleUpdates, sigMeta keybase1.SignatureMetadata) {
+	if len(stateToUpdate.inner.ActiveInvites) == 0 {
+		return
+	}
+
+	for role, uvs := range roleUpdates {
+		if role != keybase1.TeamRole_NONE {
```

Comment:
What is the thinking behind _not_ obsoleting invites if the user is being removed?

Reply:
Hm, I was on a fence with this one. Conceptually, there should not be a chain when there will be an active invite for someone that we are removing from the team, and we are only "implicitly completing" invites when client adds someone and forgets to complete it. But practically there can be. Do you think we should sweep even on `None`, for safety?

Reply:
Yeah I do think we should sweep for safety. On the principle of assuming the input may be wonked and making the best output under the circumstances